index.ts

```diff
@@ -1,82 +1,209 @@
+import { init, Ditto, Document, QueryResultItem } from "@dittolive/ditto";
+import * as readline from "readline/promises";
+import { stdin as input, stdout as output } from "node:process";
 
-import { init, Ditto, Document } from '@dittolive/ditto'
-import * as readline from 'readline/promises'
-import { stdin as input, stdout as output } from 'node:process';
+let ditto;
+let tasks: Document[] = [];
+let queryResult;
 
-let ditto
-let subscription
-let liveQuery
-let tasks: Document[] = []
+// Attachments dont to up as part of a Document.
+// Using QueryResultItem item instead.
+let attachments: QueryResultItem[] = [];
 
-async function main () {
-  await init()
+async function main() {
+  await init();
 
   ditto = new Ditto({
-    type: 'onlinePlayground',
-    appID: 'YOUR_APP_ID',
-    token: 'YOUR_TOKEN'
-  })
-  ditto.startSync()
-
-  subscription = ditto.store.collection("tasks").find("isDeleted == false").subscribe()
-  liveQuery = ditto.store.collection("tasks").find("isDeleted == false").observeLocal((docs, event) => {
-    tasks = docs
-  })
-  let isAskedToExit = false
-  
+    type: "onlinePlayground",
+    appID: "REPLACE_WITH_YOUR_APP_ID",
+    token: "REPLACE_WITH_YOUR_TOKEN",
+  });
+  ditto.startSync();
```

Comment:
```suggestion
  ditto.disableSyncWithV3();
  ditto.startSync();
```

Should always be called when mutating DQL APIs are used.